benchmark/MIcrosoft.IdentityModel.Benchmarks/identitymodel.benchmarks.yaml

```diff
@@ -0,0 +1,24 @@
+﻿jobs:
+  benchmarks:
+    source:
+      repository: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet
+      branchOrCommit: westinm/benchmark
+      project: benchmark\Microsoft.IdentityModel.Benchmarks\Microsoft.IdentityModel.Benchmarks.csproj
+    variables:
+      filterArg: "*"
+      jobArg: short
+    arguments: --job {{jobArg}} --filter {{filterArg}} --memory
+    options:
+      benchmarkDotNet: true
+    
+scenarios:
+  JsonWebTokenTests:
+    application:
+      job: benchmarks
+
+profiles:
```

Comment:
Remove this. You should only use crank locally for debugging purposes, and it's already defined in public profile files, you should import of the aspnet ones instead (standard profiles in /scenarios).